ui_framework/src/components/button/_button.scss

```diff
@@ -0,0 +1,123 @@
+// Our base button
+.kuiButton {
+  display: inline-block;
+  appearance: none;
+  cursor: pointer;
+  height: $kuiSizeXXL;
+  text-decoration: none;
+  border: solid 1px $kuiColorPrimary;
+  border-radius: $kuiBorderRadius;
+  color: $kuiColorPrimary;
+  padding: 0 $kuiSizeS;
+  min-width: $kuiSize * 7;
+  text-align: center;
+  font-family: $kuiFontFamily;
+  transition: all $kuiAnimSpeedNormal $kuiAnimSlightBounce;
+  whitespace: nowrap;
+
+  &.kuiButton--small {
+    height: $kuiSizeXL;
+  }
+
+  .kuiButton__icon {
+    vertical-align: -2px;
+    fill: $kuiColorPrimary;
+  }
+
+  &:hover, &:focus {
+    text-decoration: underline;
+    background-color: transparentize($kuiColorPrimary, .9);
+  }
+
+  &:focus {
+    transform: translateY(2px);
+  }
+
+  &.kuiButton--fill {
+    background-color: $kuiColorPrimary;
+    color: #FFF;
+    border-color: $kuiColorPrimary;
+
+    &:hover, &:focus {
+      background-color: darken($kuiColorPrimary, 5%);
+      border-color: darken($kuiColorPrimary, 5%);
+    }
+
+    .kuiButton__icon {
+      fill: #FFF;
+    }
+  }
+}
+
+.kuiButton__inner {
```

Comment:
Should we call this `.kuiButton__content`? Based on your suggestion in the Table PR: https://github.com/elastic/kibana/pull/13249#discussion_r130702909